vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -4,34 +4,27 @@ sidebarDepth: 2
+
+- [Initialization](#initialization): `ent bundle init` initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando Hub. The bundle format relies on a single JSON descriptor as the project manifest. 
 
-The `init` subcommand initializes a new bundle project, either with the default files and folders or from an existing bundle in an Entando hub. The bundle format relies on a single JSON descriptor as the project manifest. 
+- [Build](#build): With the structure established, `ent bundle build` generates micro frontend (MFE) and microservice (MS) components. These are filtered by type and name and assigned version numbers. 
 
-With the structure established, the `build` subcommand executes processes appropriate to component type and stack.Micro frontend (MFE) and microservice (MS) components are generated in parallel. They are filtered by type and name and assigned version numbers. 
+- [Run](#run): `ent bundle run` and Keycloak integration enable components to be tested locally, external to an Entando cluster.   
 
-The `run` subcommand in conjuction with Keycloak integration enables components to be tested locally, external to an Entando cluster. Multiple components are run in parallel via commands dependent on type and stack, with a log file for each individual component generated in the .entando/logs directory.  
+- [Package](#package): `ent bundle pack` constructs the artifacts and Docker images for the bundle. A single image is created for the bundle and all micro frontends while each microservice is allocated its own image. 
 
-The `pack` subcommand constructs the artifacts and Docker images for the bundle. A single image is created for the bundle and all micro frontends while each microservice is allocated its own image. These are published to a Docker repository, after which the bundle can be deployed and installed. 
+- [Publish](#publish): `ent bundle publish` pushes the Docker images to a Docker repository, after which the bundle can be deployed and installed. 
 
-Once a bundle has been published, it can be delivered to the ECR of an Entando Application via the `deploy` subcommand. The bundle custom resource is exported and tags are retrieved from Docker Hub.
+- [Deploy](#deploy): `ent bundle deploy` delivers a published bundle to the ECR of an Entando Application. The bundle custom resource is exported and tags are retrieved from Docker Hub.
 
-In the final phase of bundle creation, the `install` subcommand applies the bundle to the Entando instance. It is then available in the App Builder for unlimited reuse within the application.
+- [Install](#install): In the final phase of bundle creation, the `ent bundle install` applies the bundle to the Entando instance. It is then available in the App Builder for unlimited reuse within the application.
```

Comment:
In the final phase of bundle creation, the ent bundle ... >> In the final phase, the ent 

In the final phase of bundle creation sounds awkward and  "of bundle creation" is not necessary. This whole section is discussing only that and that is clear